dcpy/utils/geospatial.py

```diff
@@ -1,3 +1,4 @@
+# ? move this file to dcpy.builds?
```

Comment:
Good question. I think we definitely want geospatial utils (including geosupport related things), but some of the interaction (or discrete steps) are definitely going to be more build related. Good here for now, but as this gets added to and matures we'll need to think about that division for geospatial-related tasks

Reply:
and I was gonna ask in this PR if we like merging comments like this to main?

it's not a `TODO` or a `HACK`, but I made up a list in the [wiki](https://github.com/NYCPlanning/data-engineering/wiki/Operations#style-guides) for standard comment types we might like to use

Reply:
Yeah, I think this gets at something worth teasing out (not necessarily now, though). In particular, it feels a little misplaced to have product-level information like product column-names baked into our pure utils. Problem for tomorrow, though